src/ripple/app/misc/Validations.cpp

```diff
@@ -87,6 +87,16 @@ class ValidationsImp : public Validations
 private:
     bool addValidation (STValidation::ref val, std::string const& source) override
     {
+        if (val->getFieldU32 (sfLedgerSequence) <= app_.getMaxLedger())
```

Comment:
Heads up that this will need to go to `RCLValidations.cpp` after rebasing on 0.80.0-b1